### ISSUE TITLE
[BUG] Interesting and non-breaking unexpected behaviour happens when changing `machine-name` at run time without unregistering probe first

### ISSUE THREAD

**Issue by equals215** [equals215/deepsentinel#8]
Bug "fixed" in this commit : https://github.com/equals215/deepsentinel/commit/db5577ceb59cf5b8ef4a18e28590c463de28d91c

When running a server in normal conditions, plugging an agent that runs with `machine-name=localhost` then executing `config machine-name localhost2` produces this behaviour : 
```
TRAC[2024-04-08 17:27:08.332] Crafting PagerDuty provider
INFO[2024-04-08 17:27:08.332] Warming PagerDuty instance
TRAC[2024-04-08 17:27:08.772] Crafting PagerDuty provider
INFO[2024-04-08 17:27:08.773] Warming PagerDuty instance
WARN[2024-04-08 17:27:08.998] Alerting is disabled due to -no-alert flag
INFO[2024-04-08 17:27:08.999] ————————————
INFO[2024-04-08 17:27:08.999] deepSentinel API server starting...
INFO[2024-04-08 17:27:08.999] Serving on 0.0.0.0:5000
INFO[2024-04-08 17:27:08.999] Probe inactivity delay: 2s
INFO[2024-04-08 17:27:08.999] Degraded to failed threshold: 10
INFO[2024-04-08 17:27:08.999] Failed to alerted low threshold: 20
INFO[2024-04-08 17:27:08.999] Alerted low to alerted high threshold: 30
DEBU[2024-04-08 17:27:09.000] Starting monitoring.Handle

 ┌───────────────────────────────────────────────────┐
 │                 DeepSentinel API                  │
 │                   Fiber v2.52.2                   │
 │               http://127.0.0.1:5000               │
 │       (bound on host 0.0.0.0 and port 5000)       │
 │                                                   │
 │ Handlers ............. 5  Processes ........... 1 │
 │ Prefork ....... Disabled  PID ............. 42488 │
 └───────────────────────────────────────────────────┘

INFO[2024-04-08 17:50:10.460] Starting probe thread                         machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:10.460] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:10.461] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=1 status=normal
TRAC[2024-04-08 17:50:11.463] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:11.463] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=2 status=normal
TRAC[2024-04-08 17:50:12.466] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:12.466] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=3 status=normal
TRAC[2024-04-08 17:50:13.468] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:13.468] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=4 status=normal
TRAC[2024-04-08 17:50:14.469] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:14.470] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=5 status=normal
TRAC[2024-04-08 17:50:15.471] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:15.472] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=6 status=normal
TRAC[2024-04-08 17:50:16.474] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:16.474] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=7 status=normal
TRAC[2024-04-08 17:50:17.476] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:17.476] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=8 status=normal
TRAC[2024-04-08 17:50:18.486] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:18.486] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=9 status=normal
TRAC[2024-04-08 17:50:19.494] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:19.494] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=10 status=normal
INFO[2024-04-08 17:50:20.498] Deleting probe                                machine=localhost1 probe=localhost1
INFO[2024-04-08 17:50:29.852] Starting probe thread                         machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:29.854] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:29.856] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=1 status=normal
TRAC[2024-04-08 17:50:30.859] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:30.859] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=2 status=normal
TRAC[2024-04-08 17:50:31.861] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:31.861] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=3 status=normal
TRAC[2024-04-08 17:50:32.864] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:32.864] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=4 status=normal
TRAC[2024-04-08 17:50:33.865] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:33.866] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=5 status=normal
TRAC[2024-04-08 17:50:34.868] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:34.868] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=6 status=normal
TRAC[2024-04-08 17:50:35.869] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:35.869] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=7 status=normal
TRAC[2024-04-08 17:50:36.871] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:36.871] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=8 status=normal
TRAC[2024-04-08 17:50:37.873] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:37.873] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=9 status=normal
TRAC[2024-04-08 17:50:38.874] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:38.875] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=10 status=normal
TRAC[2024-04-08 17:50:39.876] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:39.876] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=11 status=normal
TRAC[2024-04-08 17:50:40.878] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:40.878] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=12 status=normal
TRAC[2024-04-08 17:50:41.879] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:41.879] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=13 status=normal
TRAC[2024-04-08 17:50:42.882] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:42.882] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=14 status=normal
TRAC[2024-04-08 17:50:43.884] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:43.884] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=15 status=normal
TRAC[2024-04-08 17:50:44.886] Received report                               machine=localhost1 probe=localhost1 status=normal
TRAC[2024-04-08 17:50:44.886] Payload stored in timeserie                   machine=localhost1 probe=localhost1 size=16 status=normal
INFO[2024-04-08 17:50:45.889] Starting probe thread                         machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:45.889] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:45.889] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=1 status=normal
WARN[2024-04-08 17:50:46.887] No payload received for 2.000283792s. Machine localhost1 is now in degraded state
TRAC[2024-04-08 17:50:46.890] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:46.890] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=2 status=normal
TRAC[2024-04-08 17:50:47.892] Received report                               machine=localhost2 probe=localhost2 status=degraded
TRAC[2024-04-08 17:50:47.892] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=17 status=degraded
INFO[2024-04-08 17:50:47.892] Machine localhost2 is back in normal state
WARN[2024-04-08 17:50:48.891] No payload received for 2.000451959s. Machine localhost2 is now in degraded state
TRAC[2024-04-08 17:50:48.893] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:48.894] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=18 status=normal
TRAC[2024-04-08 17:50:49.895] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:49.895] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=19 status=normal
TRAC[2024-04-08 17:50:50.899] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:50.899] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=20 status=normal
TRAC[2024-04-08 17:50:51.902] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:51.902] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=21 status=normal
TRAC[2024-04-08 17:50:52.905] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:52.905] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=22 status=normal
TRAC[2024-04-08 17:50:53.908] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:53.908] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=23 status=normal
TRAC[2024-04-08 17:50:54.909] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:54.909] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=24 status=normal
TRAC[2024-04-08 17:50:55.911] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:55.911] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=25 status=normal
TRAC[2024-04-08 17:50:56.914] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:56.914] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=26 status=normal
TRAC[2024-04-08 17:50:57.916] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:57.916] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=27 status=normal
TRAC[2024-04-08 17:50:58.919] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:58.919] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=28 status=normal
TRAC[2024-04-08 17:50:59.922] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:50:59.922] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=29 status=normal
TRAC[2024-04-08 17:51:00.924] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:00.924] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=30 status=normal
TRAC[2024-04-08 17:51:01.933] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:01.933] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=31 status=normal
TRAC[2024-04-08 17:51:02.936] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:02.936] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=32 status=normal
TRAC[2024-04-08 17:51:03.938] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:03.939] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=33 status=normal
TRAC[2024-04-08 17:51:04.940] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:04.940] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=34 status=normal
TRAC[2024-04-08 17:51:05.942] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:05.943] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=35 status=normal
TRAC[2024-04-08 17:51:06.945] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:06.945] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=36 status=normal
TRAC[2024-04-08 17:51:07.947] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:07.947] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=37 status=normal
WARN[2024-04-08 17:51:08.900] No payload received for 22.009464959s. Machine localhost2 is now in failed state
TRAC[2024-04-08 17:51:08.950] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:08.950] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=38 status=normal
TRAC[2024-04-08 17:51:09.952] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:09.952] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=39 status=normal
TRAC[2024-04-08 17:51:10.954] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:10.954] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=40 status=normal
TRAC[2024-04-08 17:51:11.956] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:11.956] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=41 status=normal
TRAC[2024-04-08 17:51:12.958] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:12.958] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=42 status=normal
TRAC[2024-04-08 17:51:13.960] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:13.960] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=43 status=normal
TRAC[2024-04-08 17:51:14.961] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:14.961] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=44 status=normal
TRAC[2024-04-08 17:51:15.963] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:15.963] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=45 status=normal
TRAC[2024-04-08 17:51:16.966] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:16.966] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=46 status=normal
TRAC[2024-04-08 17:51:17.969] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:17.969] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=47 status=normal
TRAC[2024-04-08 17:51:18.971] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:18.971] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=48 status=normal
TRAC[2024-04-08 17:51:19.973] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:19.973] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=49 status=normal
TRAC[2024-04-08 17:51:20.974] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:20.975] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=50 status=normal
TRAC[2024-04-08 17:51:21.977] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:21.977] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=51 status=normal
TRAC[2024-04-08 17:51:22.979] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:22.980] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=52 status=normal
TRAC[2024-04-08 17:51:23.982] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:23.982] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=53 status=normal
TRAC[2024-04-08 17:51:24.985] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:24.985] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=54 status=normal
TRAC[2024-04-08 17:51:25.987] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:25.987] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=55 status=normal
TRAC[2024-04-08 17:51:26.990] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:26.990] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=56 status=normal
TRAC[2024-04-08 17:51:27.992] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:27.992] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=57 status=normal
TRAC[2024-04-08 17:51:28.995] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:28.995] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=58 status=normal
TRAC[2024-04-08 17:51:29.997] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:29.998] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=59 status=normal
TRAC[2024-04-08 17:51:31.000] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:31.000] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=60 status=normal
TRAC[2024-04-08 17:51:32.001] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:32.002] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=61 status=normal
TRAC[2024-04-08 17:51:33.005] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:33.005] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=62 status=normal
TRAC[2024-04-08 17:51:34.022] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:34.022] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=63 status=normal
TRAC[2024-04-08 17:51:35.024] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:35.024] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=64 status=normal
TRAC[2024-04-08 17:51:36.025] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:36.026] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=65 status=normal
TRAC[2024-04-08 17:51:37.028] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:37.028] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=66 status=normal
TRAC[2024-04-08 17:51:38.030] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:38.030] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=67 status=normal
TRAC[2024-04-08 17:51:39.031] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:39.031] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=68 status=normal
TRAC[2024-04-08 17:51:40.033] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:40.034] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=69 status=normal
TRAC[2024-04-08 17:51:41.039] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:41.040] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=70 status=normal
TRAC[2024-04-08 17:51:42.041] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:42.041] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=71 status=normal
TRAC[2024-04-08 17:51:43.043] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:43.043] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=72 status=normal
TRAC[2024-04-08 17:51:44.044] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:44.044] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=73 status=normal
TRAC[2024-04-08 17:51:45.046] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:45.046] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=74 status=normal
TRAC[2024-04-08 17:51:46.048] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:46.048] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=75 status=normal
TRAC[2024-04-08 17:51:47.050] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:47.050] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=76 status=normal
TRAC[2024-04-08 17:51:48.052] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:48.052] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=77 status=normal
WARN[2024-04-08 17:51:48.917] No payload received for 1m2.025795209s. Machine localhost2 is now in alertedLow state
TRAC[2024-04-08 17:51:48.917] Alerting machine localhost2 low
WARN[2024-04-08 17:51:48.917] No low alert provider configured. Can't send alert.
TRAC[2024-04-08 17:51:49.056] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:49.060] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=78 status=normal
TRAC[2024-04-08 17:51:50.062] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:50.062] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=79 status=normal
TRAC[2024-04-08 17:51:51.064] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:51.064] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=80 status=normal
TRAC[2024-04-08 17:51:52.066] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:52.066] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=81 status=normal
TRAC[2024-04-08 17:51:53.070] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:53.070] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=82 status=normal
TRAC[2024-04-08 17:51:54.073] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:54.073] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=83 status=normal
TRAC[2024-04-08 17:51:55.075] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:55.075] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=84 status=normal
TRAC[2024-04-08 17:51:56.077] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:56.078] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=85 status=normal
TRAC[2024-04-08 17:51:57.080] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:57.080] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=86 status=normal
TRAC[2024-04-08 17:51:58.081] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:58.082] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=87 status=normal
TRAC[2024-04-08 17:51:59.084] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:51:59.084] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=88 status=normal
TRAC[2024-04-08 17:52:00.087] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:00.087] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=89 status=normal
TRAC[2024-04-08 17:52:01.088] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:01.089] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=90 status=normal
TRAC[2024-04-08 17:52:02.090] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:02.090] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=91 status=normal
TRAC[2024-04-08 17:52:03.092] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:03.092] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=92 status=normal
TRAC[2024-04-08 17:52:04.094] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:04.094] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=93 status=normal
TRAC[2024-04-08 17:52:05.097] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:05.097] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=94 status=normal
TRAC[2024-04-08 17:52:06.099] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:06.099] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=95 status=normal
TRAC[2024-04-08 17:52:07.100] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:07.100] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=96 status=normal
TRAC[2024-04-08 17:52:08.102] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:08.103] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=97 status=normal
TRAC[2024-04-08 17:52:09.104] Received report                               machine=localhost2 probe=localhost2 status=normal
TRAC[2024-04-08 17:52:09.104] Payload stored in timeserie                   machine=localhost2 probe=localhost2 size=98 status=normal
INFO[2024-04-08 17:52:10.108] Deleting probe                                machine=localhost2 probe=localhost2
WARN[2024-04-08 17:52:48.948] No payload received for 2m2.057599084s. Machine localhost2 is now in alertedHigh state
TRAC[2024-04-08 17:52:48.949] Alerting machine localhost2 high
WARN[2024-04-08 17:52:48.949] No high alert provider configured. Can't send alert.
WARN[2024-04-08 17:53:08.958] Machine localhost2 is still in alertedHigh state
^Csignal: interrupt
```

This isn't normal as probe `localhost1` shouldn't rename itself to `localhost2` but a `localhost2` probe should be created and the `localhost1` probe should degrade over time as it doesn't receive reports

**Comment by equals215** [comment]
Might be fixed by #10

**Comment by equals215** [comment]
Fixed by #10 🤠
